views/survey.pug

```diff
@@ -32,22 +32,35 @@ html
                     // details(id = detailsID onclick="countDetails(this.id)" open)= item.elaboration_example
                       summary <strong>Example</strong> (space to toggle)
                     // input(type='hidden' id=detailsInputID name=detailsID value=0)
-                  
-                  if item.response == 'text'
+              else
+                if item.group_prompt !== 'undefined'
+                  h3= item.group_prompt
+                if(item.image != "Field/15")
```

Comment:
This is formatted for javascript, and pug is not  javascript. It should probably be something more like:
```pug
if condition
```
(like the other ones here, also of course you can [find out more about pug at its docs](https://pugjs.org))

I also don't quite know why this is the right condition to be looking for. I think I'd just say: 
```pug
if item.image
```
so it checks if theres an image and renders it if its there. 

One complicating factor here is that we might need to have images go different places sometimes, so this might be an overly simple way of setting this up.